scripts/collect-typos.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+pushd "$(git rev-parse --show-toplevel)" >/dev/null 
+
+git checkout -b catch-all-typos
+
+while read -r pr; do
+	patch -p1 <<< "$(gh pr diff "$pr")" 
+done <<< "$(gh pr list --label "typo" | awk '{print $1}')"
+
+git add -A
+git commit -m "Squash all PR:s with the typo label."
```

Comment:
it should definitely be 1 commit (that's the point of collecting the typo PRs), but maybe we should include the authors in the commit message like : 

```
Helped-by: Foo Bar <baz@zub.com>
```

Reply:
This part should be fixed in the new PR.